app/views/match_decisions/thirteen/_hearing_outcome.haml

```diff
@@ -0,0 +1,65 @@
+.match-decision.c-card.c-card--flush.card--block
+  .c-card__content
+    = simple_form_for @decision, url: access_context.match_decision_path(@match, @decision) do |form|
+      .o-choose.o-choose--flush.o-choose--flush-top
+        - if @decision.accessible_by?(current_contact)
+          .o-choose__choice{class: [('o-choose__choice--disabled' if !@decision.editable?), ( 'o-choose__choice--100' unless can_reject_matches?)]}
+            %header
+              .o-choose__title
+                .c-choice-icon.c-choice-icon--confirm.mr-4
+                %h3 Continue Match
+            .o-choose__content
+              - if @match.match_route.show_hearing_date
+                %p
+                  - hearing_date = @match.thirteen_hearing_scheduled_decision.criminal_hearing_date
+                  - if hearing_date.present?
+                    = "Please note: #{Translation.translate('a criminal background hearing has been schedule for')}"
+                    %strong
+                      = "#{hearing_date.to_s}."
+                = form.input :criminal_hearing_outcome_recorded, as: :boolean, required: true, inline_label: "#{Translation.translate('The outcome of the criminal background hearing has been recorded')}"
```

Comment:
If these are new translations, let's drop the words `criminal background` so these are just **hearings**.  It's fine that it's in the variable name, but I'm pretty sure these aren't always criminal hearings.  (This appears elsewhere as well, and should probably just be `hearing` or `hearings` with a translation.